bfxapi/websockets/generic_websocket.py

```diff
@@ -57,15 +57,7 @@ async def send(self, data):
             await self.ws.send(data)
 
 def _start_event_worker():
-    def start_loop(loop):
-        asyncio.set_event_loop(loop)
-        loop.run_forever()
-    event_loop = asyncio.new_event_loop()
-    ee = EventEmitter(scheduler=asyncio.ensure_future)
-    worker = Thread(target=start_loop, args=(event_loop,))
-    worker.daemon = True
-    worker.start()
-    return ee
+    return EventEmitter(scheduler=asyncio.ensure_future)
```

Comment:
No need for the event loop anymore?

Reply:
Nope I decided to get rid of it entirely. The drawback of this is, once an event is triggered, the logic within the function will be executed on the same thread as the websocket (like the nodejs implementation). But I thought it would be better to leave the multi processing shinanigans to the end user to handle. If they are doing CPU intensive tasks then they can pass the data onto a separate thread themselves once an event is triggered.

Reply:
Much better solution :+1: